src/web/mjs/connectors/TRWebtoon.mjs

```diff
@@ -0,0 +1,15 @@
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
+
+export default class RawMangaUpdate extends MangaReaderCMS {
```

Comment:
The classname does not match the name of the website, i suggest to change the classname.

Reply:
sry, forget to change the class name